pkg/await/extensions_ingress.go

```diff
@@ -402,7 +475,7 @@ func (iia *ingressInitAwaiter) errorMessages() []string {
 }
 
 func (iia *ingressInitAwaiter) checkAndLogStatus() bool {
-	success := iia.ingressReady && iia.checkIfEndpointsReady()
+	success := iia.ingressReady && iia.checkIfEndpointsReady() && len(iia.eventError) == 0
```

Comment:
I'm hesitant to include the event errors as part of the Ingress check given that this is specific to the GKE Ingress Controller, and not all Ingress Controllers on GCP are the same, let alone across cloud providers. I defer to only showing events for logging/debugging purposes for now.

As an aside, many GKE users tend to **not** use the GKE Ingress Controller because it is [costly](https://github.com/kubernetes/ingress-gce/blob/master/BETA_LIMITATIONS.md#static-and-ephemeral-ips). In my personal experience it costs about ~$20 per month per ingress managed, so folks tend to look at self-managing NGINX Ingress Controller as a better, and cheaper means.